.github/workflows/ci_skipped.yaml

```diff
@@ -2,11 +2,15 @@ name: CI Skipped
 on:
   pull_request:
     branches: ["**"]
-    paths:
-      - 'README.md'
-      - 'LICENSE'
-      - "**/README.md"
-      - "**/docs/**"
+    paths-ignore:
+      - '**/*.rs'
+      - '**/*.toml'
+      - '**/*.yaml'
```

Comment:
One `.yaml` was a `.yml` I already changed it in [this PR](https://github.com/lambdaclass/ethrex/pull/1259) but maybe we want to add it just in case it happens again.